alert/alert.css

```diff
@@ -0,0 +1,47 @@
+@import url("https://fonts.googleapis.com/icon?family=Material+Icons");
+@import url("/style.css");
+.material-icons {
+  margin: 0 0.5rem;
+  cursor: pointer;
+}
```

Comment:
I am not sure, but I guess it's not a good practice to play with the original (library) class. You could rather add a new class to make the necessary changes. Check it with others also.

Reply:
Ohh thanks buddy. I didn't knew that